src/uint/boxed/rand.rs

```diff
@@ -36,9 +36,19 @@ impl RandomBits for BoxedUint {
 impl RandomMod for BoxedUint {
     fn random_mod<R: RngCore + ?Sized>(rng: &mut R, modulus: &NonZero<Self>) -> Self {
         let mut n = BoxedUint::zero_with_precision(modulus.bits_precision());
-        random_mod_core(rng, &mut n, modulus, modulus.bits());
+        random_mod_core(rng, &mut n, modulus, modulus.bits())
+            .expect("RngCore used as TryRngCore failed");
```

Comment:
This should be a `map_err(|_: Infallible| panic!())` to protect this assumption against a refactor.

Reply:
You can use `let Ok(()) = random_mod_core(rng, &mut n, modulus, modulus.bits());`. This feature was [stabilized](https://blog.rust-lang.org/2024/10/17/Rust-1.82.0.html#omitting-empty-types-in-pattern-matching) in Rust 1.82, so we can use it.

Reply:
TIL